packages/fast-components-react-msft/src/utilities/index.ts

```diff
@@ -1,3 +1,7 @@
 const DisplayNamePrefix: string = "MSFT";
 
 export { DisplayNamePrefix };
+
+export function isNullOrUndefined(value: any): boolean {
```

Comment:
instead of adding this, can we just us `isNil` from lodash? We already bring it in as a peer dependency.

https://lodash.com/docs#isNil

Reply:
removed `isNullOrUndefined` and used `isNil` from lodash